woocommerce-services.php

```diff
@@ -282,14 +284,22 @@ public static function get_wcs_admin_style_url() {
 			}
 		}
 
-		function wpcom_static_url( $file ) {
+		public function wpcom_static_url( $file ) {
 			$i   = hexdec( substr( md5( $file ), -1 ) ) % 2;
 			$url = 'http://s' . $i . '.wp.com' . $file;
 			return set_url_scheme( $url );
 		}
 
 		public function __construct() {
 			$this->wc_connect_base_url = self::get_wc_connect_base_url();
+			add_action(
+				'before_woocommerce_init',
+				function() {
+					if ( class_exists( '\Automattic\WooCommerce\Utilities\FeaturesUtil' ) ) {
+						\Automattic\WooCommerce\Utilities\FeaturesUtil::declare_compatibility( 'custom_order_tables', 'woocommerce-services/woocommerce-services.php' );
```

Comment:
Before merging, please remove the incompatibility declaration introduced in #2594